src/to_wiki.js

```diff
@@ -118,23 +168,48 @@ const serializer = new WikiTextSerializer({
    },
 
    text(state, node) {
-      let marks = node.marks || []
+      let marks = node.marks.map(m => m.type.name)
```

Comment:
We decided to ignore this since `mark` refers to a name on every line except this one, so it's clear enough as-is.